### ISSUE TITLE
Transcribe piano RH or LH part to corresponding staff

### ISSUE THREAD

**Issue by epoudrier** [polyrhythm-project/polyrhythm-website#62]
There are a number of examples where the RH of the piano is notated in the bass staff. This is problematic for group attribution when the two hands belong to a different rhythmic group.

Here is a example of the problem from R129, Janacek:
![Screen Shot 2020-01-23 at 3 58 38 PM](https://user-images.githubusercontent.com/15526880/73034290-47dc6a80-3df9-11ea-9fa6-9582e3219a27.png)

Here the piano RH needs to be assigned to the treble staff above so that the rhythmic groups can be properly divided.

**Comment by craigsapp** [comment]
This one is not strictly assigned to me, an RA one way or the other can do this sort of fix 😉 .  There are two ways to solve this problem:

(1) Fix the Sibelius file so that the cross-staff voice is placed on the treble staff (and displayed on the treble staff).  If following the original score (which I have not seen), there should be no visible rest in the top staff first measure, and the notes are in the top staff, but should be moved to the bottom staff as cross-staff notes:
      https://www.youtube.com/watch?v=HL_ARBVQcYs&t=135
But of course in this example, just displaying the notes in the top staff will be fine (it will depend on how faithful to the original notation you want to be).

(2) Fix the Humdrum file in the same manner.  In this case the music can be placed on the treble staff and optionally displayed on the bottom staff.

Which one will depend on if you want to have the Sibelius/PDF/MusicXML data closely match to the Humdrum/MEI/Verovio versions of the data.

I fixed this example manually in the past, but current work to fix systematic errors in the MusicXML to Humdrum conversion has removed all manual fixes (so it will be good not to do manual fixes for a while as I fix the errors that can be handled by software).

Here is the basic fix in Humdrum/VHV:

<img width="1392" alt="Screen Shot 2020-01-24 at 00 37 05" src="https://user-images.githubusercontent.com/3487289/73055461-b513ee00-3e41-11ea-8ef6-f24454d52b9e.png">

An alternate style is to have the notes in question housed on the top staff, but display them on the bottom staff (in which case the groupings remain correct):

<img width="1392" alt="Screen Shot 2020-01-24 at 01 00 27" src="https://user-images.githubusercontent.com/3487289/73056874-ee9a2880-3e44-11ea-9ba1-78ba05bf8973.png">


Although there is a bug in the placement of the slur in verovio at the moment.

**Comment by epoudrier** [comment]
If we cannot do the manual edits for now, the the alternate style is preferable as we are working on getting to a "'definitive' grouping for analyses purposes right now. Will this alternate style be affected by fixing systematic errors in the MusicXML to Humdrum conversion?

**Comment by craigsapp** [comment]
Yes there can be confusions, but if we make note of problem cases, these can be checked in detail later.

**Comment by epoudrier** [comment]
The best solution for R129 appears to be to edit the Sibelius file and replace all three formats (sib, xml, pdf).

@aaronearlgraham thedesired transcription as shown above, with piano RH in treble staff.

**Comment by epoudrier** [comment]
We are able to do group attribution without re-transcription. We will hold on re-transcription for this type of "error" for now.
